### DISCUSSION TITLE
Storybook dependencies for webcomponents

### DISCUSSION THREAD

**Discussion by chithiraiselvan** [storybookjs/storybook#13417] [Help]
Hi Team,

We are building webcomponents library using litelements. We are using storybook for documentation purpose. Just add these dev dependencies "@storybook/addon-essentials": "6.0.21","@storybook/addon-docs": "6.0.21", "@storybook/web-components": "6.0.21".

After that, it is added around 700 files. Any options to include required for web components related to dependencies?  Do any options reduce dependency files? Please guide me !!

**Comment by shilman** [comment]
We're actively working on performance right now, and part of that will be to reduce dependencies.

There's also a variant of Storybook maintained by @daKmoR @LarsDenBakker and the awesome folks at Open WC that's already been optimized for web components, and you should definitely check out:

https://open-wc.org/docs/demoing/storybook/#usage
